README.md

```diff
@@ -110,6 +118,24 @@ Netdata works with tons of applications, notifications platforms, and other time
   <a href="https://hub.docker.com/r/netdata/netdata"><img src="https://registry.my-netdata.io/api/v1/badge.svg?chart=dockerhub.pulls_sum&divide=1000&precision=1&units=k&label=docker+hub+pulls&after=-86400&group=incremental-sum&label=docker%20hub%20pulls%20today&options=unaligned&v44" alt="Docker Hub pulls today"></a>
 </p>
 
+### Infrastructure view
+
+Due to the distributed nature of the Netdata ecosystem, it is recommended to setup not only not one Netdata Agent on your production system, but also an additional Netdata Agent acting as a [Parent](https://learn.netdata.cloud/docs/agent/streaming). A local Netdata Agent (child), without any database or alarms, collects metrics and sends them to another Netdata Agent (parent). The same parent can collect data for any number of child nodes and serves as a centralized health check engine for each child by triggering alerts on their behalf.
+
+![Netdata Cloud](https://user-images.githubusercontent.com/423236/205926887-43024984-6d38-46ad-96cb-d0c388117c6d.png)
+
+Get started by [signing in](https://app.netdata.cloud/?utm_source=website&utm_content=top_navigation_sign_up) to Netdata.cloud and follow the setup guide.
```

Comment:
```suggestion
Get started by [signing in](https://app.netdata.cloud/?utm_source=website&utm_content=top_navigation_sign_up) to Netdata Cloud and follow the setup guide.
```

Reply:
I deliberately put netdata.cloud to promote the website

Reply:
I think we could to like @andrewm4894 suggests, have [Netdata Cloud](https://www.netdata.cloud/)

Reply:
I think Netdata Cloud sounds better too.